app_scraping/migrations/0004_auto_20191112_1202.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.6 on 2019-11-12 12:02
```

Comment:
Generally you want a single migration per pull request/merge. Multiple migrations aren't bad, but it's unnecessary churn. The simplest method to getting a single migration is:

1. Reverse all migrations. This will delete your local data: `./manage.py app_scraping zero`
2. Delete the new migration files.
3. Create the migration again as a single migration: `./manage.py makemigrations`

Reply:
Done in commit 516946b